clayui.com/gatsby/createPages.js

```diff
@@ -95,17 +95,20 @@ const createDocs = (actions, edges, mdx, blacklist = []) => {
 					const blacklistSlugs = blacklist.map(
 						({
 							node: {
-								fields: {sibling},
+								fields: {siblings},
 							},
-						}) => sibling
+						}) => siblings
 					);
 
 					createPage({
 						component: template,
 						context: {
-							blacklist: blacklistSlugs,
+							blacklist: blacklistSlugs.reduce(
+								(acc, val) => acc.concat(val),
+								[]
+							),
 							markdownJsx: mdx,
-							sibling,
+							siblings,
```

Comment:
It seems that this is coming `undefined` in some moments and we need it to be at least an empty array, apparently, it is still creating pages that are ignored for being siblings of other pages, you have to improve the `find` of `hasSibling`:

```js
const hasSibling = blacklist.find(
	({
		node: {
			fields: {siblings},
		},
	}) => siblings.includes(slug)
);
```

this seems to be the problem with errors on the console.

Reply:
Oh, that's a good catch @matuzalemsteles , it's been a couple weeks, but I think that at the time I focused too much on the queries and forgot about this file.